lib/getMainExpParameters.m

```diff
@@ -177,7 +177,7 @@
 % added F0s-amplitude because the relative dB set in volume adjustment in
 % PychPortAudio will be used in the mainExp
 cfg.volumeSettingSound = repmat(makeStimMainExp(ones(1,16), cfg,...
-    cfg.gridIOIs(end), cfg.F0s(end),cfg.F0sAmp(end)), 2,1); 
+    cfg.gridIOIs(end), cfg.F0s(end), cfg.F0sAmpGain(end)*cfg.baseAmp ), 2,1); 
```

Comment:
Don't we multiply with the `cfg.baseAmp` within `makeSequence` function? If you changed within the function multiplication, here is OK!